nextflow.config

```diff
@@ -13,20 +13,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-nextflow.enable.dsl = 2
+nextflowVersion = '!>=22.11'
+
+plugins {
+    id 'nf-validation@1.1.3'
+}
 
 params {
-    query = null
-    name = null
-    
-    species = null
-    seq_type = null
-    ref_seq = null
+    query = ""
+    species = ""
+    seq_type = ""
+    refseq = false
 
-    genes = null
+    cluster_tool = ""
```

Comment:
Is this a typo? I do not see that in any other part of MEvoLib, and I would like to know if I got to change the GUI.

Reply:
Whilst we have focused most of our work around the `genes` division, there are other 3 options provided in MEvoLib: naive rows, naive cols and PRD. The third one may be obsolete by now, but the other 2 could still be useful in some scenarios, so it may be good to set our infrastructure to plan with those in mind.

_Short answer:_ yes, I believe we should include this in the GUI as well (at the moment only `genes` will be accepted).

Reply:
Please, then put a dictionary with the tools as in the other modules, so that the GUI does not have them handmade written.

Reply:
There is already a `cluster.get_tools()` function like in the other modules (in MEvoLib). Or are you asking for something different?

Reply:
I did not see it, my bad...